include/seqan3/range/views/minimiser_hash.hpp

```diff
@@ -0,0 +1,158 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Mitra Darvish <mitra.darvish AT fu-berlin.de>
+ * \brief Provides seqan3::views::minimiser_hash.
+ */
+
+#pragma once
+
+#include <seqan3/range/views/kmer_hash.hpp>
+#include <seqan3/range/views/minimiser.hpp>
+
+namespace seqan3::detail
+{
+//!\brief views::minimiser_hash's range adaptor object type (non-closure).
```

Comment:
```suggestion
//!\brief seqan3::views::minimiser_hash's range adaptor object type (non-closure).
```
I think otherwise the links don't work